.github/workflows/auto-review-comment.yml

```
@@ -0,0 +1,34 @@
+name: Auto Review Comment
+
+on:
+  workflow_run:
+    workflows:
+      - Diff Compile
+    types:
+      - completed
+
+permissions:
+  pull-requests: write
+
+jobs:
+  auto-review:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
```

```suggestion
      - name: Checkout the repository
```

Let’s use the same name